addons/account/tests/test_account_move_send.py

```diff
@@ -1081,3 +1081,24 @@ def test_no_sending_method_selected(self):
         wizard.action_send_and_print()
         self.assertTrue(invoice.is_move_sent)
         self.assertTrue(invoice.invoice_pdf_report_id)
+
+    def test_get_sending_settings(self):
+        invoice = self.init_invoice("out_invoice", amounts=[1000], post=True)
+        wizard = self.create_send_and_print(invoice)
+        
+        expected_results = {
+            'sending_methods': ['email'],
+            'invoice_edi_format':False,
+            'extra_edis': [],
+            'pdf_report': self.env['ir.actions.report'].search([('name', '=', 'PDF')]),
+            'author_user_id': self.env.user.id,
+            'author_partner_id': self.env.user.partner_id.id,
+            'mail_template': self.env['mail.template'].search([('name', '=', 'Invoice: Sending')]),
+            'mail_lang': 'en_US',
+            'mail_body': wizard.mail_body,
+            'mail_subject': 'company_1_data Invoice (Ref INV/2019/00001)',
+            'mail_partner_ids': [invoice.partner_id.id],
+            'mail_attachments_widget': [{'id': 'placeholder_INV_2019_00001.pdf', 'name': 'INV_2019_00001.pdf', 'mimetype': 'application/pdf', 'placeholder': True}],
+        }
+        results = wizard._get_sending_settings()
+        self.assertDictEqual(results, expected_results)
```

Comment:
Not an error and I'm not asking to change that but in case you didn't know:

You can call `assertEqual` on 2 dict, it's doing the same than `assertDictEqual` :)
[Source](https://docs.python.org/3/library/unittest.html#:~:text=The%20list%20of%20type%2Dspecific%20methods%20automatically%20used%20by%20assertEqual()%20are%20summarized%20in%20the%20following%20table.%20Note%20that%20it%E2%80%99s%20usually%20not%20necessary%20to%20invoke%20these%20methods%20directly.)